docs/design/autoware-interfaces/components/sensing.md

```diff
@@ -0,0 +1,46 @@
+# Sensing
+
+![Node diagram](images/Sensing-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### The sensing module contains the driver for the sensor, so there is no input
+
+## Output
+
+### Camera image
+
+Image data from camera. Used by the Perception.
+
+- [sensor_msgs/Image](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Image.html)
+
+### Lidar original pointcloud
+
+Pointcloud without preprocessing. Used by the Perception.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Lidar filtered pointcloud
+
+Pointcloud after preprocessing. Used by the Perception.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Radar pointcloud
+
+Pointcloud from radar. Used by the Perception.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Corrected IMU data
+
+IMU data with yaw rate offset compensated. Used by the Localization.
+
+- [sensor_msgs/Imu](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Imu.html)
+
+### Vehicle pose
```

Comment:
Which sensor data are you referring? (Maybe mistook the vehicle velocity?)

Reply:
The vehicle velocity information of the vehicle comes from the wheel speed sensor. Do you mean I should add this sensor?

Reply:
Yes that's also my intention! So I have two questions:
- Which sensor are you referring to in "Vehicle pose"?
- Why did you excluded wheel speed sensor (that you've mentioned above)?

Reply:
My fault. I thought the IMU data had already obtained the vehicle velocity information, just like angular_velocity.
I will add this topic in my next submission

Reply:
The vehicle direction is from the steering wheel angle sensor, but the data of this sensor is directly sent to the control module by the vehicle chassis, but not the sensing module

Reply:
I think vehicle direction is not included in steering wheel angle sensor. The steering angle is the different value from the vehicle direction (which is the yaw angle in the map coordinate). 

So seems there are two options? Let me hear your thought as well @YamatoAndo @yukkysaito @aohsato 

### A. Exclude vehicle information from this page
- Camera image
- ...
- Corrected IMU data

### B. Include vehicle sensor data as well
- Camera image
- ...
- Corrected IMU data
- Vehicle sensor data (steering angle & wheel odometry velocity)

Reply:
My previous statement is not clear.
What I would like to say is that the steering wheel angle sensor is a sensor related to the direction of the vehicle, but not related to the sensing module. Therefore, the information about this sensor is not mentioned in the document.
The vehicle pose is provided by the IMU.

Reply:
> What I would like to say is that the steering wheel angle sensor is a sensor related to the direction of the vehicle, but not related to the sensing module. Therefore, the information about this sensor is not mentioned in the document.
Understood! Thanks for the clarification.

> The vehicle pose is provided by the IMU.
I don't think so. IMU only publishes angular velocity and acceleration in Autoware. (And furthermore, acceleration is currently not used)

Reply:
I think it is better to describe the vehicle speed and steering information provided by the vehicle CAN in the vehicle interface page.
It is better to use only external sensors for the information described here.

Reply:
I took a look at my first version of the diagram( [here](https://raw.githubusercontent.com/autowarefoundation/autoware-documentation/82c16372fded0ffc982f84a39aa7cffaea2af953/docs/design/autoware-interfaces/components/images/Sensing-Bus-ODD-Architecture.drawio.svg) ). The sensor on the diagram corresponds to the output below. Here we can see that the pose refers to the initial position provided by GNSS。
So should we add which sensor the data comes from to the description of each output topic?

Reply:
Good idea! I will change this in the next submission